.devcontainer/arch-linux/devcontainer.json

```diff
@@ -19,7 +19,24 @@
             "source": "/tmp/.X11-unix",
             "target": "/tmp/.X11-unix",
             "type": "bind"
+        },
+        {
+            "source": "arch-linux-persist-conan",
```

Comment:
I think the volume names should have names that associate them with Artery, e.g. "artery-conan-arch-linux".